### ISSUE TITLE
Repeated mj-tag's inside mj-attributes get ignored

### ISSUE THREAD

**Issue by rinopo** [mjmlio/mjml#1194]
Hi,

Repeated `mj-tag`s inside the `mj-attributes` seem to be ignored except for the last ones.

**Reproduction Steps:**

Use following MJML sources:

https://mjml.io/try-it-live/HkEN2o8Az
```html
<mjml>
  <mj-head>
    <mj-attributes>
      <mj-all color="blue" />
      <mj-all font-size="40px" />
    </mj-attributes>
  </mj-head>
  <mj-body>
    <mj-section>
      <mj-column>
        <mj-text>
          Hello World!
        </mj-text>
      </mj-column>
    </mj-section>
  </mj-body>
</mjml>
```
or

```html
<mjml>
  <mj-head>
    <mj-attributes>
      <mj-text color="blue" />
      <mj-text font-size="40px" />
    </mj-attributes>
  </mj-head>
  <mj-body>
    <mj-section>
      <mj-column>
        <mj-text>
          Hello World!
        </mj-text>
      </mj-column>
    </mj-section>
  </mj-body>
</mjml>
```

**Expected behavior:**

Both `color` and `font-size` attributes are applied to `Hello World!`.

**Observed behavior:**

Only the last attribute (here `font-size`) is applied to `Hello World!`.

**MJML version:**

The "try-it-live" version (4.0.5?).

**Comment by rinopo** [comment]
You could work around this like:

```html
    <mj-attributes>
      <mj-text color="blue" font-size="40px" />
    </mj-attributes>
```

But I thought I'd file it as an issue since:

- Repeated `mj-tag`s inside `mj-attributes` used to work back in MJML3.
- It is not clearly documented (allowed or not allowed).
- I prefer using repeated `mj-tag`s because I find it more readable, espeially when you want to specify more than a couple of attributes.

Thank you so much for your consideration.

**Comment by iRyusa** [comment]
Thanks for finding this @rinopo 

We'll try to fix it for 4.1 release. cc @kmcb777

**Comment by kmcb777** [comment]
Hi @rinopo we released a new beta for mjml 4.1, which you can install with `npm install mjml@next`, could you check if it resolves the problem you had ?
